### DISCUSSION TITLE
Do you need to add common components to the template ?

### DISCUSSION THREAD

**Discussion by misitebao** [misitebao/wails-template-vue#29] [Polls · 投票]
Recently, some users have proposed adding some common components to the template, such as forms, tables, etc., so that developers can start development directly, especially for those back-end developers who are not familiar with front-end engineering. I fully understand the original intention of these proposals. However, I haven't decided whether to add these content for the time being, because I think this project is just a Vue-based template, it only provides the most basic content, the rest should be left to the developers themselves, because everyone's needs are different , the added content must be guaranteed to be available to everyone. If the default content is included, then users who don't need it will need to delete it themselves after initializing the project, I'm not sure if that's an additional burden. So posting a vote here.

> 近来，有用户提议在模板中增加一些常用组件，比如表单、表格等，这样便于开发者可以直接上手开发，特别是对于那些不熟悉前端工程的后端开发者。我充分理解这些提议的初衷。不过我暂时并没有确定是否要加入这些内容，原因是我认为此项目只是一个基于Vue的模板，它仅仅提供最基础的内容，其他的应该交给开发者自己去做，因为每个人需求不一样，加入的内容必须保证是所有人都可以用到的。如果加入了默认内容，那么那些不需要此内容的用户在初始化项目后则需要自己删除这些内容，我不确定这是否是一个额外的负担。所以在此处发出一个投票。

**Comment by eighteenzheng** [comment]
I don't think these contents should be added. As the subject said, the template should only provide the contents that everyone can use. The front-end part of this project is a **standard front-end engineering project**. If developers do not want to use the default style components, they can Use CSS to beautify it yourself, or add your favorite component library to the project. To do this, even if you are a developer who is not familiar with front-end engineering, you can also find a complete tutorial in the corresponding component library documentation.

**Reply by zhangya4548** [reply]
不会前端的后端就不是这样想的, 我就是想找个能快速开发客户端工具的框架,  里面有完整的表单页, 列表页组件,  我只关心我功能实现即可,不比考虑烦人的各种前端问题
